packages/server/src/middleware/nuxt.js

```diff
@@ -134,20 +134,24 @@ const getCspString = ({ cspScriptSrcHashes, allowedSources, policies, isDev, isR
   }
 
   if (policyObjectAvailable) {
-    const transformedPolicyObject = transformPolicyObject(policies, cspScriptSrcHashes)
+    const transformedPolicyObject = transformPolicyObject(policies, cspScriptSrcHashes, isDev)
```

Comment:
What do you think adding `'unsafe-eval'` to `cspScriptSrcHashes` if it doesn't exist in dev mode ? Then it should work for both string and object cases.